### PR TITLE
"Change <dbname> logo"

### PR DESCRIPTION
**Pull request by Embobada**
Changing logo Embobadawiki

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -5096,6 +5096,7 @@
 		'dwplivewiki' => "//$wmgUploadHostname/dwplivewiki/c/c0/Logo_135.png",
 		'emulationwiki' => "//$wmgUploadHostname/emulationwiki/3/3b/Wiki_logo.png",
 		'enmarchewiki' => "//$wmgUploadHostname/enmarchewiki/a/af/Logo-em-alsace68.png",
+		'embobada' => "//$wmgUploadHostname/embobadawiki/3/35/Schermafbeelding_2017-05-11_om_16.37.12.png",
```
Comment (Reception123) [inline LocalSettings.php]: It has to be "embobadawiki" with the suffix at the end

Comment (Embobada) [inline LocalSettings.php]: Thank you

**Comment by Reception123** [comment]
@Embobada Are you fine editing it or would you like me to?

**Comment by Reception123** [comment]
Redone in #1539

**Pull request closed without merging**